basic/src/test/java/org/jboss/hal/testsuite/test/configuration/webservices/configuration/WebServicesClientConfigurationTestCase.java

```diff
@@ -0,0 +1,225 @@
+package org.jboss.hal.testsuite.test.configuration.webservices.configuration;
+
+
+import org.apache.commons.lang.RandomStringUtils;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.hal.testsuite.creaper.ResourceVerifier;
+import org.jboss.hal.testsuite.fragment.config.webservices.WebServicesConfigurationPropertyWizard;
+import org.jboss.hal.testsuite.fragment.config.webservices.WebServicesConfigurationWizard;
+import org.jboss.hal.testsuite.test.configuration.webservices.WebServicesTestCaseAbstract;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+@RunWith(Arquillian.class)
+@RunAsClient
+public class WebServicesClientConfigurationTestCase extends WebServicesTestCaseAbstract {
+
+    @AfterClass
+    public static void afterClass() throws CommandFailedException, IOException, TimeoutException, InterruptedException {
+        try {
+            administration.reloadIfRequired();
+            administration.restartIfRequired();
```

Comment:
Is the restart really necessary here? Try to run tests on fresh instance of server and the check if `restart-required` state is set.